python_modules/dagster-graphql/dagster_graphql/implementation/fetch_assets.py

```diff
@@ -98,14 +98,26 @@ def get_asset(graphene_info, asset_key):
     return GrapheneAsset(key=asset_key, definition=asset_node)
 
 
-def get_asset_events(graphene_info, asset_key, partitions=None, limit=None, before_timestamp=None):
```

Comment:
prefer that you break this out into `get_materialization_events` / `get_observation_events`.  Do we anticipate a time where we'd fetch both together?  It's fine for this to be broken out as a helper function, but being explicit will reduce future confusion.

Reply:
I don't anticipate we'd fetch both together without having to filter them into separate lists. I've broken this out into two separate methods.